teams/3/teams.html

```diff
@@ -0,0 +1,383 @@
+              <svg>svg</svg>
+              <span>span</span>
+            </a>
+          </li>
+          <li>
+            <a href="#">nav item</a>
+          </li>
+          <li>
+            <a href="#"
+              >nav item <i><svg></svg></i>
+            </a>
+          </li>
+          <li>
+            <a href="#"
+              >nav item <i><svg></svg></i
+            ></a>
+          </li>
+          <li>
+            <a href="#">nav item</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <button>Login</button>
+        <button>Sign Up</button>
+      </div>
+    </header>
+    <main>
+      <section class="feature">
+        <h2>H2</h2>
+        <span>span</span>
+        <p>p</p>
+        <button>We’re hiring!</button>
+      </section>
+      <section class="hiring">
+        <h2>H2</h2>
+        <span>span</span>
+        <p>p</p>
+        <ul class="employees">
+          <li>
+            <figure>
+              <figcaption>
+                <img src="" alt="Picture" />
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <figcaption>
+                <img src="" alt="Picture" />
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <figcaption>
+                <img src="" alt="Picture" />
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <figcaption>
+                <img src="" alt="Picture" />
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+
+          <li>
+            <figure>
+              <img src="" alt="Picture" />
+              <figcaption>
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="" alt="Picture" />
+              <figcaption>
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="" alt="Picture" />
+              <figcaption>
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="" alt="Picture" />
+              <figcaption>
+                <h3>H3</h3>
+                <h4>H4</h4>
+              </figcaption>
+            </figure>
+          </li>
+        </ul>
+      </section>
+      <section class="core-values">
+        <span>span</span>
+        <h2>h2</h2>
+        <p>p</p>
+        <ul class="values">
```

Comment:
Keep each value inside of an article instead of ul and li.

Reply:
The same reason here as I wrote above. The listname is Core-values and then there is a list of values. So we decided to go with unordered list. Please tell us more, why do you think so?

Reply:
The article contains standalone content that can exist on its own. So semantically it makes more sense to use an article here than a simple ul li based structure.